framework/requirements/YiiRequirementChecker.php

```diff
@@ -155,7 +155,7 @@ function render()
         if (!isset($this->result)) {
             $this->usageError('Nothing to render!');
         }
-        $baseViewFilePath = dirname(__FILE__) . DIRECTORY_SEPARATOR . 'views';
+        $baseViewFilePath = __DIR__ . DIRECTORY_SEPARATOR . 'views';
```

Comment:
Should not be modified like that because requirements checker requirements are lower than framework's. It should be able to say that requirements aren't met.